### PR TITLE
IAP-3616 - Remove 'Customize application sharing settings' section

### PR DESCRIPTION
**Pull request by teivanov**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/en/docs/apim_administration/apiportal_admin/customize_page_content.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ When the setting is set to **No**, an application developer cannot see the detai
 
 When the setting is set to **Yes** or **No** an organization administrator can see all the users that an application is shared with, regardless of who shared it with them.
 
+{{< alert title="Note" color="primary" >}}From API Portal 7.7 November 2020 **Enable user listing** option is removed and only organization administrators will see the sharing tab on application edit page.{{< /alert >}}
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apiportal_admin/customize_page_content.md:154]: This is a release specific information, which should be added to the release notes page, not to the main docs. For more information, see https://techweb.axway.com/confluence/display/RDAPI/Do+not+add+specific+release+changes+to+the+latest+docs

Comment (jowens-axway) [reply content/en/docs/apim_administration/apiportal_admin/customize_page_content.md:154]: @teivanov @andreamussap Teo should the whole section here be removed from the main docs as there will no longer be a way to customise this as we are removing the "Enable user listing" from the JAI?
https://deploy-preview-1364--axway-open-docs.netlify.app/docs/apim_administration/apiportal_admin/customize_page_content/#customize-application-sharing-settings
Also we need to add a note on this to the release notes for Nov.

Comment (teivanov) [reply content/en/docs/apim_administration/apiportal_admin/customize_page_content.md:154]: @jowens-axway @andreamussap It's a little bit tricky here. Our documentation is for the 7.7 version in general, so I think "Enable user listing" section should stay for those who use API Portal version prior to the November release. Andrea has a good point that this Note I've added is release-specific information and we should add it in the release notes. However if we include it in the November release notes and leave this section without a note, in future customers installing January **2021** for example won't know why in the documentation there's an option which is not available on their instance of API Portal. Not sure what's the best decision here.

**Review by andreamussap — APPROVED** [review]
On **develop** branch -

https://deploy-preview-1364--axway-open-docs.netlify.app/docs/apim_administration/apiportal_admin/customize_page_content/

**Comment by andreamussap** [comment]
Updates as per email "Discussion - Do not add specific release changes to the latest docs"

* Created KB article with 'Customize application sharing settings' section - https://support.axway.com/en/admin/kbadmin/article-details/id/181283
* Remove 'Customize application sharing settings' section from 77 docs.
* @jowens-axway  to create a PR to update APIP Release notes Nov20 to add this information to 'Removed features' sections, including a link to the KB article.

**Pull request merged**